modules/loader-utils/src/lib/library-utils/library-utils.js

```diff
@@ -36,12 +36,19 @@ export function getLibraryUrl(library, moduleName, options) {
     return modules[library];
   }
 
+  // Load from local files, not from CDN scripts in Node.js
+  // TODO - needs to locate the modules directory when installed!
+  if (!isBrowser) {
+    return `modules/${moduleName}/dist/libs/${library}`;
+  }
+
+  // In browser, load from external scripts
   if (options.CDN) {
     assert(options.CDN.startsWith('http'));
     return `${options.CDN}/${moduleName}@${VERSION}/dist/libs/${library}`;
   }
 
-  // Note - Worker loading requires paths relative to worker script "location"?
+  // TODO - loading inside workers requires paths relative to worker script location...
   if (isWorker) {
     return `../src/libs/${library}`;
```

Comment:
same here. To me, it is kind of fragile to resolve a path like this way.

Reply:
#600